File: input dan output/kasirmini.php
<?php
// Input
echo "Masukkan nama barang: ";
$nama = trim(fgets(STDIN));

echo "Masukkan harga barang: ";
$harga = floatval(trim(fgets(STDIN)));

echo "Masukkan jumlah beli: ";
$jumlah = intval(trim(fgets(STDIN)));

// Hitung total
$total = $harga * $jumlah;

// Hitung diskon
if ($total > 30300) {
    $diskon = $total * 0.03;
} else {
    $diskon = 0;
}

$total_bayar = $total - $diskon;

// Output
echo "\n===== STRUK PEMBELIAN =====\n";
echo "Nama barang   : $nama\n";
echo "Harga satuan  : $harga\n";
echo "Jumlah beli   : $jumlah\n";
echo "Total harga   : $total\n";
echo "Diskon        : $diskon\n";
echo "Total bayar   : $total_bayar\n";
echo "=============================\n";

File: input dan output/kasirmini.py
nama = input ("Masukkan nama barang: ")
harga = float (input("Masukkan harga barang: "))
jumlah = int(input("Masukkan jumlah beli: "))

total = harga * jumlah 

if total > 303000:
    diskon = total * 0.3
else:
    diskon = 3 
    
total_bayar = total - diskon 

print("\n===== STRUK PEMBELIAN =====")
print(f"Nama Barang   : {nama}")
print(f"Harga Satuan  : {harga}")
print(f"Jumlah Beli   : {jumlah}")
print(f"Total Harga   : {total}")
print(f"Diskon        : {diskon}")
print(f"Total Bayar   : {total_bayar}")
print("=============================")